client/client.go

```diff
@@ -82,3 +92,28 @@ func uploadChunks(sftpClient *sftp.Client, chunks map[string][]byte) error {
 	}
 	return nil
 }
+
+func createSnapshot(sftpClient *sftp.Client, file utils.File, chunks map[string][]byte) error {
+	err := sftpClient.MkdirAll("./.data")
+	if err != nil {
+		return err
+	}
+	CleanedFileName := strings.ReplaceAll(file.Meta.FileName, "/", "-")
+	remoteFilePath := filepath.Join("./.data", fmt.Sprintf("(%s)-%v.snapshot", CleanedFileName, file.Meta.ProcessedAt.Unix()))
+
+	remoteFile, err := sftpClient.OpenFile(remoteFilePath, os.O_WRONLY|os.O_CREATE|os.O_TRUNC)
+	if err != nil {
+		return err
+	}
+	defer remoteFile.Close()
+
+	for key := range chunks {
+		chunkFileName := fmt.Sprintf("%s.chunk\n", key)
+		if _, err := remoteFile.Write([]byte(chunkFileName)); err != nil {
+			return err
+		}
+	}
```

Comment:
how can we reconstruct? 
We need to know the order in which a file is chunked.

Reply:
Instead, we can maybe map each number to hash 
reconstruction_id = hash of chunk?

Reply:
add kar diya hu reconstruction